### ISSUE TITLE
Inexplicable characters appear in vgm_stat

### ISSUE THREAD

**Issue by denjhang** [vgmrips/vgmtools#28]
![截图未命名0124234016](https://user-images.githubusercontent.com/61811583/214339777-f6a428ea-8a56-4f03-a85d-e88220b748fa.jpeg)
[tag2.zip](https://github.com/vgmrips/vgmtools/files/10491642/tag2.zip)
All files are in zip

**Comment by ValleyBell** [comment]
I can't seem to reproduce this with the files you sent, but I know that vgm_stat has trouble with emojis and any other Unicode character that doesn't fit into 16 bits.
I'll have a look at this at some point.

**Comment by ValleyBell** [comment]
I just noticed that you're using an outdated version. Please update to the last release from 2022.
The bug you got doesn't happen anymore. (I think it was fixed by d61b4656ee4c9704f84481c470a62b2f45c0ff99.)

**Comment by ValleyBell** [comment]
vgm_stat reads the Unicode VGM tags, but converts it down to your system's local encoding. That is a lossy process.
It then does all the formatting on *that* converted text.

Fixing this requires a major write, so I'm afraid you will have to just accept that question mark for now and manually fix that line.

At some point I'll get around to rewrite it to apply all formatting to the Unicode text. That should resolve the internal encoding issues and leave only the Windows console itself to cause trouble.

(There is a precompiled version from 2022-12-30 on vgmrips, btw. That one even aligns the 1-hour times properly.)

**Comment by denjhang** [comment]
> vgm_stat reads the Unicode VGM tags, but converts it down to your system's local encoding. That is a lossy process. It then does all the formatting on _that_ converted text.
> 
> Fixing this requires a major write, so I'm afraid you will have to just accept that question mark for now and manually fix that line.
> 
> At some point I'll get around to rewrite it to apply all formatting to the Unicode text. That should resolve the internal encoding issues and leave only the Windows console itself to cause trouble.
> 
> (There is a precompiled version from 2022-12-30 on vgmrips, btw. That one even aligns the 1-hour times properly.)

I tried version 2022.12.30 and there is still a redundant "?".

**Comment by ValleyBell** [comment]
Yes, that is because this character can not be represented with your system's 8-bit codepage. (That's what I meant with "local encoding".)

You will have to accept that question mark and fix it manually until I do a major rewrite of vgm\_stat.
